resources/analysisTools/snvPipeline/snvCalling.sh

```diff
@@ -66,8 +71,23 @@ filenameMPileupError="${MPILEUP_SUBDIR}/${MPILEUPOUT_PREFIX}${PID}.${chr}.error"
 
 FILENAME_VCF_SNVS_TEMP=${FILENAME_VCF_SNVS}.tmp
 
-#[[ ! -f $filenameMPileupTemp ]] && 
-${BCFTOOLS_BINARY} mpileup ${MPILEUP_OPTS} -r ${chr} -f ${REFERENCE_GENOME} $tumorbamfullpath | ${BCFTOOLS_BINARY} call ${BCFTOOLS_OPTS} > $filenameMPileupTemp
+## For variant calling from ALT and HLA region in the GRCh38 genome
+# The *_CONTIGS_FILE contains the list of the contigs names
+MPILEUP_REGION_OPTION=""
+if [[ "$chr" == "HLA" ]]
+then
+    MPILEUP_REGION_OPTION="-R ${CHROMOSOME_HLA_CONTIGS_FILE}"
```

Comment:
I'm a bit confused. samtools/0.1.19 says

```
-R           ignore RG tags
```

That is, it does not take a parameter. The help also says that there are only BAMs as input files. I don't see (from the docs) why this should work.